### PR TITLE
I18n categories

### PR DESCRIPTION
**Pull request by sgiaccio**

### CODE REVIEW

File: common/i18n/resources/en.js [hunk 1]
```diff
@@ -184,4 +192,38 @@ module.exports = {
...
+    level: 'Level',
+    items: 'Items',
+    confirmDelete: 'Delete the item with all children? $t(common.cantUndoWarning)',
+    deleteItem: 'Delete item',
+  },
+
+  jobs: {
+    item: 'Item',
+    errors: 'Errors',
+
```
Comment (minotogna) [inline common/i18n/resources/en.js]: should item and error plus plural go to common? what do you think?

File: webapp/loggedin/surveyViews/items/itemsTable.js [hunk 2]
```diff
@@ -21,7 +24,7 @@ const TableRow = props => {
       <div className="name">
         {name}
         <ErrorBadge validation={item.validation}/>
-        <WarningBadge show={!item.usedByNodeDefs} label="Unused"/>
+        <WarningBadge show={!item.usedByNodeDefs} label={i18n.t('unused')}/>
```
Comment (minotogna) [inline webapp/loggedin/surveyViews/items/itemsTable.js]: is the key maybe ```itemsTable.unused``` ?

Comment (sgiaccio) [reply webapp/loggedin/surveyViews/items/itemsTable.js]: Yes, fixed

File: common/i18n/resources/en.js (hunk 1 above)
Comment (sgiaccio) [reply common/i18n/resources/en.js]: yes, better

**Pull request merged**